medusa/storage/s3_compat.py

```diff
@@ -0,0 +1,202 @@
+# -*- coding: utf-8 -*-
```

Comment:
Could you rename this file to `s3_base_storage` instead? While it's not the case everywhere, I think it would be good to standardize file names with their main class name from now on like we would in Java.
Also `base` sounds more accurate since it also serves as the base for the "real" AWS S3 implementation.

Reply:
Sure